packages/lib/src/Base.svelte

```diff
@@ -2,25 +2,16 @@
   // @ts-ignore
   import libstyles from './app.css?inline';
   import { getApi } from './api';
-  import cs from './locales/cs/default.json';
-  import en from './locales/en/default.json';
-  import fi from './locales/fi/default.json';
-  import it from './locales/it/default.json';
-  import nb from './locales/nb/default.json';
-  import pl from './locales/pl/default.json';
-  import sv from './locales/sv/default.json';
+  import initTransaltor from './i18n';
 
+  let {
+    lang
+  }: { lang?: string  } = $props();  
+  const htmlLang = document.querySelector('html')?.getAttribute('lang');
+  
   const apiPromise = getApi();
 
-  let translator = {
-    cs,
-    en,
-    fi,
-    it,
-    nb,
-    pl,
-    sv
-  };
+  const translator = initTransaltor(lang || htmlLang || 'en')
```

Comment:
So, this will use the language set on the component and then fall back to the language of the html tag and finally to english.

Reply:
Looks good.

`initTransaltor` > `initTranslator`